src/processor/mapper/map_hash_join.cpp

```diff
@@ -40,8 +40,15 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapLogicalHashJoinToPhysical(
     auto hashJoin = (LogicalHashJoin*)logicalOperator;
     auto outSchema = hashJoin->getSchema();
     auto buildSchema = hashJoin->getChild(1)->getSchema();
-    auto buildSidePrevOperator = mapLogicalOperatorToPhysical(hashJoin->getChild(1));
-    auto probeSidePrevOperator = mapLogicalOperatorToPhysical(hashJoin->getChild(0));
+    std::unique_ptr<PhysicalOperator> probeSidePrevOperator;
+    std::unique_ptr<PhysicalOperator> buildSidePrevOperator;
```

Comment:
Leave a comment here that we need to map the side into which the SIP is passed first.